2_serving_natively_with_amazon_sagemaker/deploy.ipynb

```diff
@@ -34,6 +34,8 @@
     "role = sagemaker.get_execution_role()\n",
     "\n",
     "#This is the fine-tuned roberta-base transformer hosted on an S3 bucket.\n",
+    "# Have we considered a bring-your-own-model example instead of using a pre-trained model?\n",
+    "# That way, we can also showcase the model archiver tool of TorchServe and how we expect model artifacts to be packaged for deployment on SageMaker\n",
```

Comment:
Yes. In an earlier lab for this workshop we have them use the model archiver tool: https://github.com/tescal2/TorchServeOnAWS/blob/master/1_torchserve-install-and-inference/torchserve-install-and-inference.ipynb